src/components/TimelineEntry.vue

```diff
@@ -138,6 +138,9 @@ export default {
 	.timeline-entry:hover {
 		background-color: #F5F5F5;
```

Comment:
If you simply use the variable `var(--color-background-hover)` here, they are both standardized and you don’t need the extra rule for the .dark class. :)

Reply:
That would, however, require me to know what I was doing :wink: sorted, should be good now! Thanks :smile: